src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2Spec.scala

```diff
@@ -260,184 +254,6 @@ class UserRoutesV2Spec extends AnyFlatSpec with Matchers with TimeMatchers with
     }
   }
 
```

Comment:
Since the `getSamUserCombinedState()` method moved from `UserRoutesV2` to `UserService`, all of the following tests moved from here to the new `SupportSummarySpec`.